<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="m_pDataTypeComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QtCharts::QChartView" name="m_pChartView" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="m_pAddPushButton">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTreningi">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="m_pActionQuit"/>
   </widget>
   <addaction name="menuTreningi"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_pActionQuit">
   <property name="text">
    <string>Zako≈Ñcz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
